bcdaworker/worker/alr_test.go

```diff
@@ -119,9 +119,12 @@ func (s *AlrWorkerTestSuite) TestProcessAlrJob() {
 	err := s.alrWorker.ProcessAlrJob(ctx, s.jobArgs[0])
 	// Check Job is processed with no errors
 	assert.NoError(s.T(), err)
-    err = s.alrWorker.ProcessAlrJob(ctx, s.jobArgs[1])
-    // Check Job is processed with no errors
-    assert.NoError(s.T(), err)
+	err = s.alrWorker.ProcessAlrJob(ctx, s.jobArgs[1])
+	// Check Job is processed with no errors
+	assert.NoError(s.T(), err)
+	err = s.alrWorker.ProcessAlrJob(ctx, s.jobArgs[1])
+	// Check Job is processed with no errors
+	assert.NoError(s.T(), err)
```

Comment:
It looks like lines `122` and `125` are referencing the same index. If so is this a redundant?

Reply:
ooh good catch! Leftover from a merge - will remove the duplicate.